src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -119,15 +120,21 @@ def validate_import_key(key):
 
 
 def validate_import_feature(feature):
-    if feature:
-        if '%' in feature:
-            logger.warning("Ignoring invalid feature '%s'. Feature name cannot contain the '%%' character.", feature)
-            return False
-    else:
-        logger.warning("Ignoring invalid feature ''. Feature name cannot be empty.")
+    try:
+        validate_feature_flag_name(feature)
+        return True
```

Comment:
>return True [](http://example.com/codeflow?start=8&length=11)

Can we move the `return true` statement outside try/catch flow? It just makes the code easier to read to see a default return value if everything succeeds.

```
try {...}
except {...}
return True
```